internal/cluster/cluster.go

```diff
@@ -917,6 +920,55 @@ func (m *Manager) PrepareForInstallation(ctx context.Context, c *common.Cluster,
 	return err
 }
 
+func isEligibleForUpload(cluster *common.Cluster) bool {
```

Comment:
Can you do this in the DB query instead of in code?

Reply:
Added a db query back in the original UploadEvents function! How does it look? https://github.com/openshift/assisted-service/pull/4880/files#diff-ecbdbfaa3e0752590ccb7b5c0387c308de661ba622120129edf662c02fa1ddadR944-R946